erratatool.yml

```diff
@@ -30,6 +30,7 @@ synopsis:
   image: "OpenShift Container Platform 4.12.z images update"
   extras: "OpenShift Container Platform 4.12.z extras update"
   metadata: "OpenShift Container Platform 4.12.z OLM Operators metadata update"
+  microshift: "Red Hat build of MicroShift 4.12.z rpm"
```

Comment:
Let's do the standard "bug fix and enhancement update" lingo. Otherwise it looks good to me!